src/Microsoft.DotNet.Build.Tasks/PackageFiles/PerfTesting.targets

```diff
@@ -4,57 +4,94 @@
   <PropertyGroup>
     <TraceEventPackage>Microsoft.Diagnostics.Tracing.TraceEvent\$(TraceEventPackageVersion)</TraceEventPackage>
   </PropertyGroup>
-  
+
   <ItemGroup>
     <TraceEventNativePath Include="$(PackagesDir)\$(TraceEventPackage)\lib\native\**\*.*" />
   </ItemGroup>
-  
+
   <Target Name ="PublishPerfRunner" Condition="'$(Performance)'=='true'" BeforeTargets="RunTestsForProject">
```

Comment:
Suggestion:  Along with cleaning up the property groups, I'd suggest moving all the Targets into the same part of the file and commenting above them (note that position can affect property definitions, so be careful there, but that makes it easier to find/understand.)